os-hardening/minimal_access.sls

```diff
@@ -1,7 +1,7 @@
 {% from "os-hardening/map.jinja" import hardening with context %}
-{% for dir in hardening.read_only_folders %}
-{{dir}}:
+hardening-read-only-folders:
   file.directory:
+    - names: {{ hardening.read_only_folders }}
```

Comment:
names does not exist as an argument to ``file.directory``: http://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.directory